storm-server/src/main/java/org/apache/storm/stats/StatsUtil.java

```diff
@@ -830,6 +830,9 @@ private static TopologyPageInfo postAggregateTopoStats(Map task2comp, Map exec2n
         Map<String, Map<K, Double>> ret = new HashMap<>();
 
         Map<String, Map<K, List>> expands = expandAveragesSeq(avgSeq, countSeq);
+        if (expands == null) {
```

Comment:
this is not needed since `expandAveragesSeq` will not return null

Reply:
I can remove if desired. I would prefer to be safe.

Reply:
I am fine with it